src/helper/site-utils.php

```diff
@@ -113,6 +113,29 @@ function get_site_info( $args, $site_enabled_check = true, $exit_if_not_found =
 	return $site_data;
 }
 
+/**
+ * Populate basic site info from db.
+ *
+ * @param array $domains       Array of all domains.
+ *
+ * @return string $preferred_challenge Type of challenge preffered.
+ */
+function get_preferred_ssl_challenge(array $domains) {
+
+	$preferred_challenge = get_config_value( 'preferred_ssl_challenge', '' );
+	if ( ! empty ( $preferred_challenge ) ) {
```

Comment:
Do not directly return if value found in config. `$preferred_challenge` can also be `http` and we do not want to return `http` if any alias domains has wildcard.

Reply:
Fixed